tests/test_special.py

```diff
@@ -33,7 +33,7 @@ def setUpClass(cls):
             list(client.execute_query('CREATE FUNCTION {0}() \
                                        RETURNS TABLE AS RETURN (select 1 as number);'
                                       .format(cls.function)))
-            list(client.execute_query('CREATE LOGIN {0} WITH PASSWORD=\'yoloC123445!\''
+            list(client.execute_query('CREATE LOGIN {0} WITH PASSWORD=\'PLACEHOLDER\''
```

Comment:
have you verified the tests? please let me know if you need help with that

Reply:
I couldn't run them locally

Reply:
feel free to merge. I'll fix the tests